cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -738,11 +738,19 @@ private predicate exprNodeShouldBeIndirectOperand(IndirectOperand node, Expr e,
   not convertedExprMustBeOperand(e)
 }
 
+private predicate exprNodeShouldBeIndirectOutNode(IndirectArgumentOutNode node, Expr e) {
+  exists(CallInstruction call |
```

Comment:
Thinking about this some more: Should this only hold when call targets a constructor?

Reply:
I think that should be true already because `ConstructorCall` will always target a constructor, but I can add the constraint.

Reply:
I might just be blind, but where is the ConstructorCall constraint currently?

Reply:
It's in `ConstructorCall`'s charpred